.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Describe the content of the PR and links to related issues, bugs of features.
+
+## Requirements
+
+Before submitting this PR, please make sure:
+
+- [ ] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
+- [ ] The code builds cleanly without new errors or warnings
+- [ ] The code passes all the existing tests
+- [ ] All relevant documentation has been updated or added
```

Comment:
Since this hasn't been a common practice up to now, I think it's worth explicitly pasting in from the full checklist. Of course there can still be exceptions but this is one that's likely to be left out simply because the author didn't think of it.
```suggestion
- [ ] All relevant documentation has been updated or added
- [ ] Unit tests are added or updated for non-stencil code changes
```